src/github.sh

```diff
@@ -5,36 +5,49 @@ GITHUB_API_HEADER="Accept: application/vnd.github.v3+json"
 github::calculate_total_modifications() {
   local -r pr_number="${1}"
   local -r files_to_ignore="${2}"
+  local -r ignore_line_deletions="${3}"
+
+  local additions=0
+  local deletions=0
 
   if [ -z "$files_to_ignore" ]; then
     local -r body=$(curl -sSL -H "Authorization: token $GITHUB_TOKEN" -H "$GITHUB_API_HEADER" "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/pulls/$pr_number")
 
-    local -r additions=$(echo "$body" | jq '.additions')
-    local -r deletions=$(echo "$body" | jq '.deletions')
+    additions=$(echo "$body" | jq '.additions')
 
-    echo $((additions + deletions))
+    if [ "$ignore_line_deletions" != "true" ]; then
+      ((deletions += $(echo "$body" | jq '.deletions')))
+    fi
   else
     local -r body=$(curl -sSL -H "Authorization: token $GITHUB_TOKEN" -H "$GITHUB_API_HEADER" "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/pulls/$pr_number/files?per_page=100")
 
-    local changes=0
-
     for file in $(echo "$body" | jq -r '.[] | @base64'); do
-      local ignore_file=0
-      for file_to_ignore in $files_to_ignore; do
-        if [ -z "$file_to_ignore" ]; then
-          continue
-        fi
-        if [[ "$(jq::base64 '.filename')" == $file_to_ignore ]]; then
-          ignore_file=1
+      _jq() {
+        echo ${file} | base64 -d | jq -r ${1}
+      }
```

Comment:
Why not reusing [`jq::base64`](https://github.com/CodelyTV/pr-size-labeler/blob/bfc42c990b778f5dcedd44470f9ac055ae1f8c61/src/misc.sh#L29-L31) instead?

Reply:
Oh shoot, didn't realize this helper was already available. Will use!